src/Tribe/Attendee_Registration/Modal.php

```diff
@@ -89,37 +46,7 @@ function append_modal_ar_template( $unused_content, $template_obj ) {
 		// If they're not using the new views, include v1 and bail.
 		if ( ! tribe_tickets_new_views_is_enabled() ) {
 			$this->append_modal_ar_template_v1( $unused_content, $template_obj );
```

Comment:
This looks cool! to stop using markup in the BE for v2. We need to make sure that v1 for the modal is working perfectly well.

Reply:
Yep, we'll need to get to testing both sides, this just sets things up to get to that point.